Firestore/core/src/remote/serializer.cc

```diff
@@ -716,11 +716,14 @@ google_firestore_v1_Target_QueryTarget Serializer::EncodeQueryTarget(
   }
 
   if (target.start_at()) {
-    result.structured_query.start_at = EncodeBound(*target.start_at());
+    result.structured_query.start_at =
+        EncodeCursor(target.start_at()->position(),
+                     /*before*/ target.start_at()->inclusive());
```

Comment:
This should be `/* before= */` so that the linter will warn us if we do change the variable name. Here and below.